src/components/UI/Button/Button.js

```diff
@@ -1,13 +1,39 @@
-import React from 'react';
-import { Button as ButtonComponent } from 'react-native';
-
+import React from "react";
+import { TouchableOpacity, StyleSheet, Text } from "react-native";
+const styles = StyleSheet.create({
+  primary: {
+    marginTop: 6,
+    padding: 8,
+    width: 70,
+    backgroundColor: "#2fb432",
+    borderBottomRightRadius: 5,
+    borderTopRightRadius: 5,
+  },
+  secondary: {
+    marginTop: 6,
+    padding: 8,
+    width: 122,
+    backgroundColor: "#2fb432",
+  },
+  textStyle: {
+    color: "white",
+    fontWeight: "bold",
+    textAlign: "center",
+    fontSize: 20,
+  },
+});
 const Button = (props) => {
+  const styleArray = [];
+  if (props.primary) {
+    styleArray.push(styles.primary);
+  }
+  if (props.secondary) {
```

Comment:
use if else if format for primary and secondary classes in future as we dont want to apply some wiered styles